molgenis-settings-ui/pom.xml

```diff
@@ -76,7 +76,7 @@
                         </goals>
                         <configuration>
                             <skip>${skip.js}</skip>
-                            <arguments>--frozen-lockfile</arguments>
+                            <arguments>--mutex network --frozen-lockfile</arguments>
```

Comment:
What does `--mutex network` do?

Reply:
Prevents parallel execution of yarn install, which breaks the cache.